build-and-run

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [ -n "$1" ]
+then
+  git checkout "$1"
+fi
+
+echo "Building frontend..."
+
+pushd "frontend" > /dev/null 2>&1
+npm run build
```

Comment:
You could use [prefix](https://docs.npmjs.com/cli/v10/using-npm/config#prefix): `npm run build --prefix frontend`

Reply:
Fixed in [769b1e1](https://github.com/kiesraad/abacus/pull/429/commits/769b1e18f953ee4f1e9266276f0da144adcbd91b)